Podfile.lock

```diff
@@ -23,7 +23,7 @@ DEPENDENCIES:
   - SwiftLint
 
 SPEC REPOS:
-  https://cdn.cocoapods.org/:
+  trunk:
```

Comment:
I think the CDN is the right one. Something tells me you have an outdated setting in your Cocoapod installation.

Reply:
that's odd, tried with `pod repo remove trunk` and pod install on fresh project. trunk gets added to the repo list every time

Reply:
will remove this change for now to get this change in first, will take a look at this later when doing the release commit

Reply:
I believe this changes back & forth based on the local workstation's cocoapods setup, but it should be innocuous as long as we're all on sufficiently recent versions of CocoaPods (> 1.8.4 IIRC).